conclave-common/build.gradle

```diff
@@ -18,3 +18,24 @@ dependencies {
     testImplementation "org.assertj:assertj-core:$assertj_version"
     testImplementation "org.junit.jupiter:junit-jupiter:$junit_jupiter_version"
 }
+
+jar {
+    manifest {
+        attributes(
+                "Conclave-GraalVM-Version": conclave_graal_version.toString(),
```

Comment:
This is only needed by the plugin, so for better clarity it should be set in there.

Reply:
As discussed offline, I have added a comment (and a TODO).
If we do not pass the classloader to the `getManifestAttribute` function, only the manifest of the classloader module will be accessible.
We could readd the classloader parameter of the manifest or find other solution